bin/worker.php

```diff
@@ -22,7 +22,8 @@
 
     $input = new ArgvInput;
 
-    $output = new ConsoleOutput(OutputInterface::VERBOSITY_NORMAL, true);
+    $isDecorated = $workerArgv->getParameterOption('--colors', 'always') !== 'never';
+    $output = new ConsoleOutput(OutputInterface::VERBOSITY_NORMAL, $isDecorated);
```

Comment:
Worker processes were hardcoded to use decorated output (true), causing --colors=never to be ignored in parallel mode. Now respects the --colors parameter from command line arguments.